modules/gapi/include/opencv2/gapi/infer/ie.hpp

```diff
@@ -132,6 +141,7 @@ template<typename Net> class Params {
               , {}
               , {}
               , {}
+              , {}
```

Comment:
jeez

Reply:
It must be reorganized, I'm afraid of touching it, since it'll cause lot's of changes.